math/mathcore/test/testVectorizedTMath.cxx

```diff
@@ -1,6 +1,6 @@
 #include <cmath>
 #include <random>
-#include "VectorizedTMath.hxx"
+#include "../src/VectorizedTMath.cxx"
```

Comment:
Only headers should be included in other files, we cannot expect people to include a source files. There will be duplicate symbol errors in this case.

Reply:
Yes, I agree.
Corrected now.Using `inline` in the header file means that the function will have to be defined there itself. Hence I removed the `inline`.